app/views/posts/_post.html.erb

```diff
@@ -0,0 +1,44 @@
+<%= link_to user_post_path(post.author, post) do %>
+  <article class="post_view">
+    <div class="post__container">
+      <h3><%= post.title %></h3>
+
+      <% if local_assigns[:show_all_comments] %>
+        <div class="post__counters">
+          Comments: <%= post.comments_counter %>, Likes: <%= post.likes_counter %>
+        </div>
+      <% end %>
+
+      <div>
+        <%# <% if truncated_message %>
+          <%# <%= post.text.truncate(20) %> 
+        <%# <% else %> 
+          <%# <%= post.text %> 
+        <%# <% end %> 
+      <%# </div> %> 
+
```

Comment:
- [ ] For best practices and clean code, kindly make sure you remove all the commented and unused code lines from your code base